examples/analog_hamiltonian_simulation/06_Solving_SAT_with_Differential_programming.ipynb

```diff
@@ -0,0 +1,872 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.        H_cost += -1 * qml.Projector([1], wires=[i])
</pre><p>Later <code>H_cost</code> gets reassigned. Can we use a different name for the temporary variable? Maybe <code>H_vertex</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/287/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>